frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DownloadLink/style.ts

```diff
@@ -36,9 +36,6 @@ export const CodeBlock = styled.div`
   }
 `;
 
-/**
- * @deprecated by CodeBlock styles once "DOWNLOAD_UX" feature flag is removed.
- */
 export const DownloadCodeBlock = styled.div`
```

Comment:
Note `DownloadCodeBlock` styles are now used by `EmbeddingButton` component. I have subsequently removed the `@deprecated` comment.